src/components/PurchaseOrder/PO.js

```diff
@@ -90,20 +103,20 @@ class PO extends Component {
   }
 
   getData() {
-    return _.get(this.props, ['resources', 'order', 'records', 0], null);
+    return get(this.props, ['resources', 'order', 'records', 0], null);
   }
 
   onToggleSection({ id }) {
     this.setState((curState) => {
-      const newState = _.cloneDeep(curState);
+      const newState = cloneDeep(curState);
```

Comment:
Please, get rid of the `cloneDeep`. Just return the object with changed sections. React will compare states and create new one under the hood.